internal/api/impl/proxy/proxy.go

```diff
@@ -286,6 +286,15 @@ func (h *httpProxy) setupAuthentication(req *http.Request) error {
 		}
 		req.Header.Set("Authorization", fmt.Sprintf("%s %s", auth.Type, credential))
 	}
+	oauth := h.secret.OAuth
+	if oauth != nil {
+		token, err := oauth.GetToken(req.Context())
+		if err != nil {
+			return err
+		}
+		req.Header.Set("Authorization", fmt.Sprintf("%s %s", "Bearer", token.AccessToken))
```

Comment:
It's simpler like that:

```suggestion
req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", token.AccessToken))
```